src/vm/arm/cgencpu.h

```diff
@@ -15,6 +15,8 @@
 #include "utilcode.h"
 #include "tls.h"
 
+typedef BYTE byte;
```

Comment:
The file uses byte further down which is undefined, would it be better to change all instances of byte with BYTE or perhaps define this in pal_mstypes.h?